wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/zh.md

```diff
@@ -1,449 +1,331 @@
+| 旧版本不兼容 | 是 |
+| Classic 模组能够还原旧版本的功能效果 | 否 |
+| 这个更改是刻意的 | 是 |
+| 这个更改需要再三考虑 | 是 |
 
-#### 限制极限滚动速度 (SV)
+### 限制超快或超慢的滚动速度 (SV)
 
 <!-- TODO: how exactly -->
 
 一些含有变速梗（比如物件瞬移与停滞）的谱面可能看起来与预期不符，但除此之外的谱面都是可玩的。
 
-|  |  |
 | :-- | :-: |
-| 破坏向后兼容性 | 否 |
-| Classic 模组的恢复支持 | 否 |
-| 有意更改 | 是 |
-| 需要进一步考虑 | 是 |
+| 旧版本不兼容 | 否 |
+| Classic 模组能够还原旧版本的功能效果 | 否 |
+| 这个更改是刻意的 | 是 |
+| 这个更改需要再三考虑 | 是 |
 
-#### PERFECT 判定窗口期与 OD 值成正比
+### PERFECT 判定区间与 OD 值成正比
 
-无论整体难度 (OD) 设为多少，之前这个判定固定为 ±16 毫秒。
+之前，无论整体难度 (OD) 设为多少，PERFECT 判定区间固定为 ±16 毫秒。
 
-|  |  |
 | :-- | :-: |
-| 破坏向后兼容性 | 是 |
-| Classic 模组的恢复支持 | 否 |
-| 有意更改 | 是 |
-| 需要进一步考虑 | 否？ |
+| 旧版本不兼容 | 是 |
+| Classic 模组能够还原旧版本的功能效果 | 否 |
+| 这个更改是刻意的 | 是 |
+| 这个更改需要再三考虑 | 否？ |
 
-#### 转谱的打击窗口不再不同
+### 转谱的打击窗口不再与专谱不同
 
-由 osu! 模式转化到 osu!mania 的谱面的打击窗口不受 OD 值影响。
+之前，由 osu! 模式转化到 osu!mania 的谱面的打击区间不受 OD 值影响。
 
-|  |  |
 | :-- | :-: |
-| 破坏向后兼容性 | 是 |
-| Classic 模组的恢复支持 | 否 |
-| 有意更改 | 是 |
-| 需要进一步考虑 | 否？ |
+| 旧版本不兼容 | 是 |
+| Classic 模组能够还原旧版本的功能效果 | 否 |
+| 这个更改是刻意的 | 是 |
+| 这个更改需要再三考虑 | 否？ |
 
-#### 在 osu!mania 中，音频速率的调整影响打击窗口
+### 在 osu!mania 中，音频速率的调整影响打击窗口
 
 在稳定版中， osu!mania 恢复了使用 DT/HT 模组对打击窗口的影响。
 
-|  |  |
 | :-- | :-: |
-| 破坏向后兼容性 | 是 |
-| Classic 模组的恢复支持 | 否 |
-| 有意更改 | 否 |
-| 需要进一步考虑 | 是 |
+| 旧版本不兼容 | 是 |
+| Classic 模组能够还原旧版本的功能效果 | 否 |
+| 这个更改是刻意的 | 否 |
+| 这个更改需要再三考虑 | 是 |
 
-#### Flashlight 模组可视区域周围没有渐变
+### Flashlight 模组可视区域周围没有渐变
 
 <!-- TODO: comparison image  -->
 
-|  |  |
 | :-- | :-: |
-| 破坏向后兼容性 | 否 |
-| Classic 模组的恢复支持 | 否 |
-| 有意更改 | 否 |
-| 需要进一步考虑 | 是 |
-
-### 分数、表现分 (pp) 和排行榜
-
-lazer 中使用了简化过的计分系统。这套系统的主要优点是不需要播放一遍回放后才能计算分数。这套计分系统有两个可更换的计分方式：标准和经典。标准计分则将基础分数限制为 1000000 分乘以分数倍率 + 奖励分。经典计分和标准类似，但是它的分数是根据谱面中已经击打物件的数量来平方缩放过的。这两个计分方式可以在设置中切换，并且会影响游戏中所有分数显示。
-
-在每个打击物件与判定对应的分数上也存在一些差异。
-
-依赖于分数的功能，如[分数排名](/wiki/Ranking#score-ranking)和[等级](/wiki/Gameplay/Score/Total_score#level)受到的影响需要进一步讨论。
-
-lazer 中不会得到奖章（成就）。将来会在防作弊实装后启用这个功能，并且，我们也要进一步考虑由于其他游玩方面的改变，可能导致的难度变化。
-
-表现分现在还没整合到全球或特定谱面的排行榜中，而是整合到了单独版本的网站中： <https://lazer.ppy.sh/>。
-
-### 多人游戏
-
-#### 未实现 TAG co-op 模式
-
-已经计划加入此功能，但还没有开始在这个项目中工作。
-
-### 故事板
-
-#### 未实现故事板编辑器
-
-对于将要实现的内置故事板编辑器，已经有类似于稳定版中的[设计](https://www.figma.com/file/ytnnne2TH8Z956Jxhiypqq/Beatmap-Editor-2)了，但还没有开始在其上工作。
-
-#### 未实现触发器系统
-
-![](img/sb-triggers.gif)
-
-#### 对于 4:3 的故事板，谱面背景的填充模式不正确
-
-![](img/sb-bg-fill-mode.jpg)
-
-在稳定版中，谱面背景与故事板按屏幕高度填充。在 lazer 中，背景图片按屏幕宽度填充，但故事板仍按高度填充。
-
-当故事板覆盖在已存在的谱面背景上时，上述差异会导致填充模式的不一致。
-
-#### 仅限 4:3 比例的故事板未被裁剪到预期宽高比
-
-![](img/sb-crop.jpg)
-
-某些故事板被设计为只支持 4:3 宽高比，但在 lazer 中，总按 16:9 比例渲染故事板。
-
-## 缺失的外观功能
-
-计划在某个时候重新加入下列功能，但由于优先级较低暂未实现。
-
-- [Niconico](https://en.wikipedia.org/wiki/Niconico) 式的滚动回放评论（弹幕）
-- 连击提示图和相关音效
-- 倒计时
-- 光标点击波纹
-- 无法通过皮肤自定义的组件
-  - 游玩排行榜
-  - 转场跳过按钮
-  - 部分的失败/通过场景
-  - 结果屏幕
-  - 歌曲选择界面
-  - 暂停菜单
-
-## 技术差异
-
-- 开源
-- 新的用户文件存储结构
-- `无限制`刷新率现已重命名为`基本上无限制 (Basically unlimited)`，最高为 1000 Hz
-- 鼠标光标总在全屏模式中受限
-- 宽高比支持范围更广
+| 旧版本不兼容 | 否 |
+| Classic 模组能够还原旧版本的功能效果 | 否 |
+| 这个更改是刻意的 | 否 |
+| 这个更改需要再三考虑 | 是 |
```

Comment:
[also outdated](https://github.com/ppy/osu/pull/20566) I think?

But since this [issues](https://github.com/ppy/osu/issues/20559) has been solved a long time ago, I cannot guarantee that the same

Reply:
NEW PR PLZ